### DISCUSSION TITLE
Welcome to ofdmEstimator Discussions!

### DISCUSSION THREAD

**Discussion by BradyVolpe** [BradyVolpe/ofdmEstimator#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dchard** [comment]
Dear @BradyVolpe 

First of all, thanks for this tool, it is pretty good to have something like this.

In my home country, they just added a 11MHz OFDMA block to the upstream and I wanted to calculate the capacity, and this is the result I got:

<img width="1075" height="284" alt="Image" src="https://github.com/user-attachments/assets/9740688a-72fc-48bc-8bed-83682aca6a45" />

As you can see the diff between the 25 and 50KHz subcarrier spacing is almost 3 times, that cannot be right.

For comparison, I calculated a 6MHz block on 64QAM as that is the closest to the known 6.4MHz@64QAM==30.75Mbit capacity of an SC-QAM channel:

<img width="1074" height="287" alt="Image" src="https://github.com/user-attachments/assets/74a0d356-2beb-4bfa-bdfa-c4b8608bc3a0" />

Again, the 50KHz spacing gives almost 4 times more BW, while the 25KHz is the one which seems to be the realistically correct value.

When I give more BW, the difference is less and less, at 96MHz for example the diff is only 11%.

Question is if the calculator is correct on these smaller OFDMA BWs, or what can be the explanation that on lower BWs the difference is so big between 25 and 50KHz spacing?
